inst/shiny/ui.R

```diff
@@ -88,12 +88,22 @@ appCSS <-
 
 containerfunction <- function(...) {
   div(
-    style = "padding:8rem",
+    style = "padding:5rem",
     div(class = "jumbotron jumbotron-fluid",
         style = "border:solid #f7f7f9;background-color:rgba(0, 0, 0, 0.5)",
         align = "justify", ... ))
 }
 
+plotcontainerfunction <- function(...) {
+  div(
+    #style = "padding:0.1rem",
+    div(class = "front",
+        style = "border:solid #f7f7f9;background-color:rgba(0, 0, 0, 0.5);padding:1rem",
```

Comment:
The borders should be the same thickness as containerfunction. I made the plots go all the way to the edge so that we could see the largest plot possible.

Reply:
> The borders should be the same thickness as containerfunction. I made the plots go all the way to the edge so that we could see the largest plot possible.

Ah sorry, yeah, thinkness is the same but `containerfunction` has soft corners and `plotcontainerfunction` hasn't.

Reply:
I added in the placeholder and left-aligned the headers with a little more space after them.